chore: Update Grafana dashboard doc (#139)
skyscrapers/documentation: kubernetes/monitoring.md

### Example Grafana Dashboard

> You can create Grafana dashboards directly in the Grafana UI, but you can also create them as `ConfigMaps` in your Kubernetes cluster. This is useful if you want to version control your dashboards, or if you want to deploy them automatically together with your application.

Reviewer: ```suggestion
You can create Grafana dashboards directly in the Grafana UI as well as create them as `ConfigMaps` in your Kubernetes cluster. This is useful if you want to version control your dashboards, or if you want to deploy them automatically together with your application.
```
